packages/components/card/src/card.tsx

```diff
@@ -1,10 +1,28 @@
-import PropTypes from 'prop-types';
-import React from 'react';
+import React, { ReactNode } from 'react';
 import { css } from '@emotion/react';
 import { customProperties as vars } from '@commercetools-uikit/design-system';
 import { filterDataAttributes } from '@commercetools-uikit/utils';
 
-const Card = (props) => (
+type TCardProps = {
+  /**
+   * Determines the visual effect of the card. A raised card has a box shadow while a flat card has just a border.
+   */
+  type: 'raised' | 'flat';
+  /**
+   * Determines the background color of the card.
+   */
+  theme: 'light' | 'dark';
+  /**
+   * Pass a custom CSS class, useful to override the styles.
+   * <br>
+   * NOTE: This is not recommended and should only be used for building new components
+   * that require special style adjustments.
+   */
+  className?: string;
+  children?: ReactNode;
```

Comment:
These two props are optional, as they were in the original PropTypes definition.